index.rst

```diff
@@ -115,6 +115,69 @@ Use the ``--format`` option if you want to use a custom format string. Both of
 these arguments along with ``--ci`` argument run coala in non-interactive mode
 which is suitable for continuous integration.
 
+Hint
+~~~~
+
+``--json`` makes the coala output easy to read and parse. It uses
+alphabetically ordered objects, arrays, strings, values and numbers. You can
+read more about this format `here <http://www.json.org>`__.
+
+Example:
+
+.. code-block:: bash
+
```

Comment:
Hmm, maybe an example is not the way to go as this is going to make things long and hard to read